code/datums/mutations/body.dm

```diff
@@ -227,6 +227,13 @@
 	text_gain_indication = "<span class='notice'>You feel strong.</span>"
 	difficulty = 16
 
+/datum/mutation/human/stimmed
+	name = "Stimmed"
+	desc = "The user's chemical balance is more robust."
+	quality = POSITIVE
+	text_gain_indication = "<span class='notice'>You feel stimmed.</span>"
```

Comment:
Maybe add some passive self-chem purging or something to this mutation or something? It'd feel kind of lame to be working on a mutation for a while, only to roll... this.

Reply:
i'll add some more combinations but this is pretty much exactly like strength